CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Current
+
+### Fixed
+- VB Win Forms projects should keep import for 'System.Windows.Forms' [#474](https://github.com/dotnet/upgrade-assistant/pull/474)
```

Comment:
are we linking pull requests?

Reply:
Hm... seems we're not consistent. @cathysull, @sunandabalu, @twsouthwick, should we be linking issues or PRs in the change log?

I vote for PR since that's where the change actually happens, but I don't feel strongly about it.

Reply:
I'm good with either. If prefer the pr since that's the actual change, but sometimes an issue may be better as multiple prs could cover a change